x/staking/keeper/msg_server.go

```diff
@@ -99,6 +104,13 @@ func (k msgServer) CreateValidator(goCtx context.Context, msg *types.MsgCreateVa
 
 	validator.MinSelfDelegation = msg.MinSelfDelegation
 
+	globalMinSelfDelegation := k.MinSelfDelegation(ctx)
+
+	//Minimum validator self delegation must be greater than or equal to global minimum
+	if globalMinSelfDelegation.GT(validator.MinSelfDelegation) {
```

Comment:
Same, I think `validator.MinSelfDelegation.LT(globalMinSelfDelegation)` would be easier to understand, no?

Reply:
Agreed - changed in recent commit!